backend/routes/gifs.js

```diff
@@ -3,18 +3,20 @@ const { getTrendingData, getSearchedImages } = require('../services/API');
 
 router.route('/').get(async (req, res) => {
     try {
-        res.json(await getTrendingData('gifs', 25));
+        const { limit, offset } = req.query;
+
+        res.json(await getTrendingData('gifs', limit, offset));
     } catch (error) {
         res.status(404);
         console.log(error);
     }
 });
 
 router.route('/search').get(async (req, res) => {
-    const searchTerm = req.query.searchTerm;
-
     try {
-        res.json(await getSearchedImages('gif', 25, searchTerm));
+        const { searchTerm, limit, offset } = req.query;
+        
```

Comment:
Could you please remove the trailing spaces here / or remove eslint no-trailing-spaces rule. Looks like eslint throws error while building the app.